queue_services/entity-emailer/Makefile

```diff
@@ -51,9 +51,10 @@ install: clean ## Install python virtrual environment
 	pip install pip==20.1.1 ;\
 	pip install -Ur requirements.txt
 
-install-dev: ## Install local application
+install-dev: ## Install application for local development
 	. venv/bin/activate ; \
 	pip install -Ur requirements/dev.txt; \
+	pip install -Ur requirements/prod.txt; \
```

Comment:
i don't think we want to add prod.txt here.

Reply:
Totally agree!  Patrick explained the relationship between `requirements\prod.txt` and `requirements.txt`.  Apparently, we're suppose to run `make build-req` to update the requirements.txt, but because the `prod.txt` doesn't set appropriate versions this functionality is broken.